draft-ietf-tls-rfc4492bis.xml

```diff
@@ -101,7 +101,8 @@
         generation of bulk encryption/MAC keys and initialization vectors is independent of the key exchange
         algorithm and not impacted by the introduction of ECC.</t>
       <t> <xref target="tbl2"/> summarizes the new key exchange algorithms.  All of these key exchange algorithms
-        provide forward secrecy.</t>
+        provide forward secrecy if and only if fresh ephemeral keys are generated and used, and also destroyed
```

Comment:
He might, but a lot of implementations do this anyway. It protects against the attack where somebody (the government?) demands to get your private key, and that's the forward secrecy that we wanted.